gym_xiangqi/envs/xiangqi_env.py

```diff
@@ -1,13 +1,170 @@
 import gym
-# from gym import error, spaces, utils
-# from gym.utils import seeding
+from gym import spaces
+import numpy as np
+
+from gym_xiangqi.xiangqi_game import XiangQiGame
+from gym_xiangqi.piece import Piece, General, Advisor, Elephant
```

Comment:
You can make imports multi-line by having brackets I believe.

Reply:
change made based on this